packages/components/src/Typeahead/Typeahead.component.js

```diff
@@ -15,6 +15,20 @@ import {
 import { Action } from '../Actions';
 import I18N_DOMAIN_COMPONENTS from '../constants';
 
+export function getItems(items, dataFeature) {
```

Comment:
You should not export that for tests.
Render a typeahead (mount) and check the attributes